src/metabase/models/card.clj

```diff
@@ -83,7 +83,12 @@
 (defmethod mi/can-write? Card
   ([instance]
    ;; Cards in audit collection should not be writable.
-   (if (= (:collection_id instance) perms/audit-db-id)
+   (if (and
+        ;; We want to make sure there's an existing audit collection before doing the equality check below.
+        ;; If there is no audit collection, this will be nil:
+        (some? (:id (perms/default-audit-collection)))
```

Comment:
Yeah it would work, but just want to make it clear what we are checking when it is there. Also just in case perms/default-audit-collection ever gets changed back to something truthy, don't want to pass this check by mistake.

Reply:
we added this incase someone changes it back to `::noop`. What we _really_ care about is that we have an audit collection id.